cli/lib/kontena/cli/helpers/exec_helper.rb

```diff
@@ -1,87 +1,190 @@
-require_relative '../../websocket/client'
+require 'io/console'
```

Comment:
The entire `exec_helper.rb` is only loaded if you run an exec command:

```
lib/kontena/cli/containers/exec_command.rb:require_relative '../helpers/exec_helper'
lib/kontena/cli/services/exec_command.rb:require_relative '../helpers/exec_helper'
```

Or was there some even more specific purpose in only loading it when using `--interactive`?